rust/actyx/swarm/harness/src/bin/quickcheck_ephemeral.rs

```diff
@@ -142,7 +142,7 @@ fn main() {
             let mut present = OffsetMap::empty();
             let machine = sim.machines().first().unwrap();
             api.run(machine.id(), move |client| async move {
-                client.publish(PublishRequest { data: events }).await?;
+                client.0.spawn_mut(|ax| block_on(ax.publish().events(events))).await??;
```

Comment:
instead of making `.0` public, how about a `client.exec(...)` method?

Reply:
Sure but let's do it in another issue/PR, this PR is getting too complex